docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+```
+
+1. The name of the organization or user that cloned the `aws-template` and `pipeline-template` repos.
+
+2. Your AWS account ID. You can get this from the AWS console: In the navigation bar at the upper right, choose your
+    username and then copy the Account ID. It should be a 12-digit number.
+
+3. This is the ARN of the CodeStar connection to GitHub. Check out the
+    [AWS guide for setting up a CodeStar connection](https://docs.aws.amazon.com/dtconsole/latest/userguide/connections-create-github.html#connections-create-github-console),
+    then copy the ARN of your CodeStar connection here.
+
+    !!! tip
+
+        Generally it is a best practice to limit read/write access to your github account, so we recommend giving
+        CodeStar access to just your `pipeline-template` and `aws-template` repositories. You can always change this
+        later in [GitHub](https://github.com/settings/installations) if you want.
+
+### Configure your AWS profile
+
+From a terminal inside your VSCode window attached to the docker container run the following line. You may leave this
+blank aside from the region. You only need to do this once.
+
+```shell
+aws configure --profile tsdat
+# AWS Access Key ID [None]: 
+# AWS Secret Access Key [None]: 
+# Default region name [None]: us-west-2
+# Default output format [None]: 
+```
+
+Your `~/.aws/config` file should now look like this:
+
+```txt title="~/.aws/config"
+[profile tsdat]
+region = us-west-2
+```
+
+### Edit your aws credentials
+
+!!! warning
+
+    You will need to do this step ^^BEFORE^^ you deploy your stack and any time the credentials expire (usually after
+    about 12 hours).
+
+If you entered your access keys in the last step then you are good to go, otherwise open your `~/.aws/credentials` file
+and update your credentials. (1)
+{ .annotate }
+
+1. You can find your AWS credentials using the following steps:
+
+    1. Go to your AWS login page
+    2. Then click ^^PROJECT -> Administrator -> Command line or programmatic access^^  (use whatever project you are
+    an admin for)
+    3. In the section, "Option 2: Manually add a profile to your AWS credentials file (Short-term credentials)", click
+    on the box to copy the text.
+
+Your credentials file should look like this (with real values instead of the `XXXX`): (1)
+{ .annotate }
+
+1. !!! tip "Tip: Windows WSL users"
+
+        If you've already set up the AWS CLI in Windows, WSL will link to the configuration files located in the Windows
+        location. Your credentials will also be linked to the `.aws/` folder that should now be showing in your VSCode
+        Explorer tab. Edit as necessary, but note this CDK expects a profile called `tsdat` to function.
+
+        You can check that the profile was set properly using:
+        
+        `aws configure list`
+
+        If you're using a profile different from `tsdat`, set the new one using
+        
+        `export AWS_PROFILE=<new_profile>`
```

Comment:
The cdk container already sets the environment variable AWS_PROFILE=tsdat, so this step is not needed.

Reply:
Yeah, I think James recommended this because he wasn't using the container. The tutorial is tailored to using the container though, so I think it might be best to remove this WSL tip to prevent confusion.